src/arr/trove/statistics.arr

```diff
@@ -222,9 +224,15 @@ end
 
 # please see: https://online.stat.psu.edu/stat462/
 
-fun multiple-regression(x_s_s :: List<Any>, y_s :: List<Number>) -> (Any -> Number):
+fun multiple-regression(x_s_s :: List<List<Number>>, y_s :: List<Number>) -> (Any -> Number):
   doc: "multiple-regression"
-  MR.multiple-regression(x_s_s, y_s)
+  mx_X = MX.lists-to-matrix(x_s_s.map(lam(x_s): x_s.push(1) end))
+  mx_Y = MX.lists-to-matrix(y_s.map(lam(y): [list: y] end))
+  B = MX.mtx-to-list(MX.mtx-least-squares-solve(mx_X, mx_Y))
```

Comment:
Please confirm (if you haven't already!) that `mtx-least-squares-solve` computes the same results as your previous code did :)

Reply:
Yes, `multiple-regression` has been rewritten to use matrices/vectors in the latest commit. The previous test I had for it in `test-statistics.arr` continues to work.